kickstarts/centos8_build_machine.ks

```diff
@@ -0,0 +1,143 @@
+#version=RHEL8
+
+### See CHANGEME lines and adjust as needed ###
+
+ignoredisk --only-use=sda
+zerombr
+clearpart --all --initlabel --drives=sda
+partition /boot --ondisk=sda --asprimary --size=1024 --fstype=xfs
+partition pv.1 --ondisk=sda --asprimary --size=10240 --grow
+volgroup kegerator8 pv.1
+logvol swap --vgname=kegerator8 --name=swap --size=8192
+logvol / --vgname=kegerator8 --name=root --size=20480 --fstype=xfs
+logvol /build --vgname=kegerator8 --name=build --size=20480 --fstype=xfs --grow
+
+bootloader --append="rhgb quiet net.ifnames=0 biosdevname=0 crashkernel=auto" --driveorder="sda" --boot-drive=sda
+
+# Reboot after installation
+reboot
+
+# Use graphical install
+graphical
+
+# Use network installation
+url --url="http://mirror.centos.org/centos/8/BaseOS/x86_64/os/"
+repo --name="AppStream" --baseurl=http://mirror.centos.org/centos/8/AppStream/x86_64/os/
+repo --name="PowerTools" --baseurl=http://mirror.centos.org/centos/8/PowerTools/x86_64/os/
+repo --name="extras" --baseurl=http://mirror.centos.org/centos/8/extras/x86_64/os/
+repo --name="epel" --baseurl=https://mirror.atl.genesisadaptive.com/epel//8/Everything/x86_64/
+repo --name="ManageIQ-Build" --baseurl=https://copr-be.cloud.fedoraproject.org/results/manageiq/ManageIQ-Build/epel-8-x86_64/
+
+keyboard --vckeymap=us --xlayouts='us'
+
+lang en_US.UTF-8
+
+# Installation logging level
+logging --level=debug
+
+# Network information
+network --bootproto=dhcp --device=eth1 --onboot=off --noipv6 --no-activate # CHANGEME or remove based on your hardware
+network --bootproto=static --device=eth0 --gateway=192.0.2.1 --ip=192.0.2.2 --nameserver=192.0.2.1 --netmask=255.255.252.0 --noipv6 --activate # CHANGEME
+network --hostname=kegerator8.example.com # CHANGEME
+
+# Root password
+rootpw --iscrypted $1$DZprqvCu$mhqFBjfLTH/PVvZIompVP/
```

Comment:
should the encrypted password come from an environment variable or somewhere else?

Reply:
At a minimum, can we just comment what the password is right there in the comment?

Reply:
Updated the comment here and in the manageiq base kickstart